test/cram/test.t

```diff
@@ -21,28 +21,28 @@ This is a cram test for the new executable.
   
```

Comment:
We should have a test for colorless in the side-by-side view. It's currently not right.

Reply:
Not yet.

Reply:
Added additional tests